react/Title.js

```diff
@@ -3,11 +3,7 @@ import React from 'react'
 
 import styles from './searchResult.css'
 
-const Title = ({ params, showTitle }) => {
-  if (!showTitle) {
-    return null
-  }
-
+const Title = ({ params }) => {
   const title = params.term ||
```

Comment:
No risk of confusion between `Title` and `title`?

Reply:
sure, I'll rename the component